.github/workflows/build-draft-docs.yml

```diff
@@ -1,38 +1,38 @@
 name: Build Draft Guidelines
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
 jobs:
   build_docs:
     strategy:
       matrix:
         document:
-          - 'BR.md'
-          - 'EVG.md'
-          - 'NSR.md'
+          - 'BR'
+          - 'EVG'
+          - 'NSR'
```

Comment:
This was to satisfy a request from @vanbroup who found the "BR.md-hash" naming confusing, which was entirely reasonable.